zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupFileInfo.java

```diff
@@ -39,6 +38,8 @@
   private final BackupFileType fileType;
   private final long modificationTime;
   private final long size;
+  public static final long notFoundFileModificationTime = -1;
+  public static final long notFoundFileSize = -1;
```

Comment:
These are the constants. Shall we put it into constant class? At least, let's make the variable all capitalized.